EnvironmentSetup/Azure/Source/Dockerfile

```diff
@@ -1,4 +1,5 @@
 FROM ubuntu:focal
+ENV KUBECTL_VERSION='v1.23.0'
```

Comment:
Are we going with this version for Kubectl after Kubernetes version update? I remember Dev said latest version of Kubectl worked for [WAS-490](https://jira.wolfram.com/jira/browse/WAS-490). Is it not going to be same for azure also?

Reply:
The cluster version and the kubectl version differences should be +1/-1. It may work with v1.24 but I wanted to guarantee.
For more information about version differences: https://kubernetes.io/releases/version-skew-policy/

Reply:
AWS one is different because the kubectl update is related to AWS CLI v2. So I updated it to v1.24.